androdeb

```diff
@@ -204,6 +207,35 @@ function all_done_banner() {
 	c_info "All done! Run \"adeb shell\" to enter environment"
 }
 
+function check_repo_url () {
+   if [ -z $ADEB_REPO_URL ]; then
+      c_warning "Automatic download is disabled. To enable it, please set the \$ADEB_REPO_URL"
+      c_warning "environment variable as recommended in the setup instructions in the README.md"
+      do_cleanup
+      exit 0
+   fi
+}
+
+function download_headers() {
+	KERNEL_MAJOR=`$ADB shell uname -r | cut -d - -f 1 | cut -d . -f 1`
+	KERNEL_MINOR=`$ADB shell uname -r | cut -d - -f 1 | cut -d . -f 2`
+	KERNEL_VERSION="$KERNEL_MAJOR.$KERNEL_MINOR"
+	case $KERNEL_VERSION in
+		4.9) PREBUILT_HEADERS_FILE=headers-$ARCH-4.9.114.tar.gz.zip ;;
+		4.14) PREBUILT_HEADERS_FILE=headers-$ARCH-4.14.59.tar.gz.zip ;;
+		*) die 11 "Unsuported kernel version ($KERNEL_VERSION)" ;;
```

Comment:
We can make this easier for the future.
Name the prebuilt headers as "headers-$ARCH-$KERNEL_MAJOR.$KERNEL_MINOR.zip"
Then we can try to download it with curl. If it fails, then we can notify user its an unsupported kernel or the download failed.

Then if a new kernel is out, we only need to upload it and adeb will just work without any changes. We would not need to add more case statements. Does that make sense?